diff -urNp -x '*.orig' libopensmssa-0.0.9.org/Makefile.am libopensmssa-0.0.9/Makefile.am
--- libopensmssa-0.0.9.org/Makefile.am	2015-08-13 14:52:31.000000000 +0200
+++ libopensmssa-0.0.9/Makefile.am	2021-10-28 22:11:05.589241410 +0200
@@ -26,7 +26,7 @@ src_libopensmssa_la_SOURCES = src/core.c
 src_libopensmssa_la_LDFLAGS = -version-info 1 -export-dynamic \
 		$(libopensmssa_version_script)
 src_libopensmssa_la_DEPENDENCIES = $(srcdir)/libopensmssa.map
-src_libopensmssa_la_LIBADD = -lrdmacm -lpthread -L$(libdir) -losmcomp \
+src_libopensmssa_la_LIBADD = -lrdmacm -lpthread -L$(libdir) -lopensm -losmcomp \
 		$(GLIB_LIBS)
 
 man_MANS = \
